api/src/main/resources/db/migration/V1.0.0__EAS.API.sql

```diff
@@ -0,0 +1,112 @@
+CREATE TABLE ASSESSMENT_SESSION
+(
+    ASSESSMENT_SESSION_ID       UUID                                NOT NULL,
+    COURSE_SESSION              VARCHAR(6)                          NOT NULL,
+    COURSE_YEAR                 VARCHAR(4)                          NOT NULL,
+    COURSE_MONTH                VARCHAR(2)                          NOT NULL,
+    STATUS_CODE                 VARCHAR(10)                         NOT NULL,
+    ACTIVE_FROM_DATE            TIMESTAMP                           NOT NULL,
+    ACTIVE_UNTIL_DATE           TIMESTAMP                           NOT NULL,
+    CREATE_USER                 VARCHAR(32)                         NOT NULL,
+    CREATE_DATE                 TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    UPDATE_USER                 VARCHAR(32)                         NOT NULL,
+    UPDATE_DATE                 TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    CONSTRAINT ASSESSMENT_SESSION_ID_PK PRIMARY KEY (ASSESSMENT_SESSION_ID)
+);
+
+CREATE TABLE ASSESSMENT
+(
+    ASSESSMENT_ID                   UUID                                NOT NULL,
+    ASSESSMENT_SESSION_ID           UUID                                NOT NULL,
+    ASSESSMENT_TYPE_CODE            VARCHAR(10)                         NOT NULL,
+    SCHOOL_ID                       UUID                                NOT NULL,
+    STUDENT_ID                      UUID                                NOT NULL,
+    PEN                             VARCHAR(9)                          NOT NULL,
```

Comment:
I think PEN *could* be null. It will be a validation error but we still need to slam the record in the table.